src/rc_allocator.h

```diff
@@ -58,7 +58,7 @@ typedef struct rc_allocator_stats {
 typedef struct rc_allocator {
    allocator               super;
    allocator_config       *cfg;
-   buffer_handle          *bh;
+   buffer_handle           bh;
```

Comment:
The `heap_handle` field of this structure is now dead, I think.

Reply:
Took it out. And purged unnecessary `heap_handle *` variables and args in other code where `rc_allocator_init()` was being called.